### PR TITLE
Fixes #914: link logged-in user's name to MyFeeds page

### LINKED ISSUE #914: Add link to /myfeeds page to AppBar
**What would you like to be added**:

As mentioned during today's triage meeting, a button to navigate to the `/myfeeds` page needs to be added to the home page.

**Why would you like this to be added**:

No convenient means to navigate to the `/myfeeds` page currently exists. Users need to be made aware of this page's existence.

<hr>

Related: https://github.com/Seneca-CDOT/telescope/issues/889

### PR DESCRIPTION
**Pull request by Silvyre**
## Issue This PR Addresses

Fixes #914

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [x] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

This PR adds a means to navigate to the My Feeds component via a Header link. 

When a user is logged in, clicking on their username (within the Header) will now redirect them to the My Feeds page.

This change does not involve adding any additional elements to the (already-full) Header, which was a point a concern.

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by cindyledev** [comment]
To test this PR, you're gonna need to run the login docker container:
- `docker-compose up --build` for redis and elasticsearch
- `docker-compose up --build login`
- `npm start` and up open `localhost:3000`
- login using credientials
- click on user's name or the small square avatar to be redirected to `/myfeeds`

Troubleshooting:
- if `localhost:3000` shows a blank white screen, you probably need to rebuild the frontend with `npm run clean` then `npm run build`

File: src/frontend/src/components/Login/LoggedIn.jsx [hunk 1]
```diff
@@ -32,12 +33,16 @@ function LoggedIn(props) {
...
         </a>
       </Button>
-      <Typography variant="h4">| Welcome {props.name}</Typography>
-      <img
-        className={classes.avatar}
-        src={`https://unavatar.now.sh/${props.name}`}
-        alt={props.email}
-      />
+      <Link to="/myfeeds" className={classes.links}>
+        <Typography variant="h4">| Welcome {props.name}</Typography>
```
Comment (humphd) [inline src/frontend/src/components/Login/LoggedIn.jsx:37]: Let's drop `Welcome `, it's too long

File: src/frontend/src/components/Login/LoggedIn.jsx [hunk 2]
```diff
@@ -32,12 +33,16 @@ function LoggedIn(props) {
...
+      </Link>
+      <Link to="/myfeeds" className={classes.links}>
```
Comment (humphd) [inline src/frontend/src/components/Login/LoggedIn.jsx:39]: Delete this.  It is a good idea, but people don't have avatars, so it won't work.

Comment (Silvyre) [reply src/frontend/src/components/Login/LoggedIn.jsx]: Done in https://github.com/Seneca-CDOT/telescope/commit/8cab1c2bb69b1fd64b6cc7f3ab73404746dabbef

File: src/frontend/src/components/Login/LoggedIn.jsx (hunk 1 above)
Comment (Silvyre) [reply src/frontend/src/components/Login/LoggedIn.jsx]: Done in https://github.com/Seneca-CDOT/telescope/commit/8cab1c2bb69b1fd64b6cc7f3ab73404746dabbef

**Pull request merged**